lib/utils.ts

```diff
@@ -110,6 +110,24 @@ export const groupKey = (audience: Audience) =>
 
 export const lower = (s: string) => s.toLowerCase();
 
+export const getHoverGroup = (
+  a: Audience,
+  highlight: boolean,
+) => {
+  const allDOMEdges = Array.from(document.getElementsByTagName("g"));
+  const edgeIds = [
+    `${serviceKey(a)}-${groupKey(a)}-edge`,
+    `${componentKey(a)}-${groupKey(a)}-edge`,
+  ];
+  edgeIds.map((e) => {
```

Comment:
find in map + forEach could get a bit wonky with bigger service map...maybe we can derive a list of IDs and hard code a list of dom lookups using that...but we can do that later!